src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robotInverse = {};
+  const entries = Object.entries(robot);
+
+  for (let n = 0; n < entries.length; n++) {
+    for (let i = n + 1; i < entries.length; i++) {
```

Comment:
don't write loop inside another loop
This task you can do only with one loop for

Hint: for checking if key exists in the object use `hasOwnProperty`